iocore/hostdb/P_RefCountCache.h

```diff
@@ -281,6 +311,14 @@ RefCountCachePartition<C>::copy(std::vector<RefCountCacheHashingValue> *items)
   }
 }
 
+template <class C>
+void
+RefCountCachePartition<C>::metric_inc(int metric_enum, int64_t data) {
+  if (this->rsb) {
+    RecIncrGlobalRawStatCount(this->rsb, metric_enum, data);
```

Comment:
Maybe? Whats there now works (meaning the numbers are correct). When I switch to that the metrics seems all messed up, presumably because they are of type `RecRawStatSyncCount` -- is there just a RecRawStatSync (not that exact name, I looked-- can't find the equivalent).